elide-core/src/main/java/com/yahoo/elide/parsers/state/CollectionTerminalState.java

```diff
@@ -28,11 +27,7 @@
 import lombok.ToString;
 import org.apache.commons.lang3.tuple.Pair;
 
-import java.util.Collection;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Set;
+import java.util.*;
```

Comment:
yeah, intellij defaults to 5 classes in same path collapse to import class.path.*. I have updated my ide, and will add the checkstyle rule as well.